app/views/templates/rulingCoverLetterTemplate.scala.html

```diff
@@ -136,21 +136,90 @@ <h3 class="govuk-heading-m">@messages("pdf.ruling.section.ifYouDisagree.title")<
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.requestReview")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.solsTeam")
+            </p>
+            <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.1")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.2")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.3")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.4")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.5")</li>
+            </ul>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.solsTeam.cannotWrite")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText", messages("pdf.ruling.section.askForReview.solsTeam.email"), messages("pdf.ruling.section.askForReview.solsTeam.email")))
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.application.footer.solicitors.office.and.legal.services.address")
+            </p>
+            <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealForIndependentTribunal.title")</h4>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.appealImmediately")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements")
+            </p>
+            <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1")</li>
+                <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2")</li>
+            </ul>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.relevantInfo")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.appealForIndependentTribunal.page"))
+            </p>
+            <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealsAndReviews.title")</h4>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealsAndReviews.moreInfo")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.appealForIndependentTribunal.page"))
```

Comment:
This was mistakenly duplicated and displaying the wrong content. Please update to `@Html(messages("pdf.ruling.section.appealsAndReviews.page"))` thanks